code/app/src/main/resources/jacoco-agent.properties

```diff
@@ -0,0 +1 @@
+output=none
```

Comment:
Ignores jacoco output for test app - added due to the following warning seen in the logs:
`W System.err: java.io.FileNotFoundException: /jacoco.exec (Read-only file system)` 
This was introduced in jacoco 0.8.5 (ref https://github.com/jacoco/jacoco/issues/968)